ard/viz/layout.py

```diff
@@ -122,6 +123,34 @@ def plot_layout(
             landscape=False,
         )
 
+    if include_mooring_system:
+        # get the coordinates of the anchors
```

Comment:
I think we may need to have some constraints on the cable routing for the mooring line avoidance. I think FAD or another tool did this for the offshore opt paper, so we need that for real comparison. I think it would just take sharing the anchor line positions with optiwindnet